### ISSUE TITLE
xmlformat incorrectly formats variablelist terms

### ISSUE THREAD

**Issue by r0ckarong** [openSUSE/daps#464]
### Problem description

When you use xmlformat to correctly format a `variablelist` the indentation and line breaks for the `<term />` contents is done incorrectly and that content comes immediately after the opening tag. This still displays a correct table but is bad for readability.

### Expected behavior

The term tags and contents are correctly indented and broken over.

### Steps to reproduce the behavior

Format content including `variablelist` in wrong notation with xmlformat.

Initial content:
```
<variablelist>
<varlistentry><term><literal>Image</literal></term><listitem><para>Select the image your machines will be created from</para></listitem></varlistentry>
</variablelist>
```

After xmlformat:
```
         <varlistentry>
          <term><literal>Image</literal> # This should be in the next line
          </term>
          <listitem>
           <para>
            Select the image your machines will be created from
           </para>
          </listitem>
         </varlistentry>
       </variablelist>
```

Manually fixed content:
```
       <variablelist>
         <varlistentry>
          <term>
           <literal>Image</literal> #This is now formatted correctly
          </term>
          <listitem>
           <para>
            Select the image your machines will be created from
           </para>
          </listitem>
         </varlistentry>
       </variablelist>
```

**Comment by ghost** [comment]
Mh... Imo, \<term/>s should be formatted like titles, so what is incorrect in your example is only the line break just before the closing \</term>. I've opened PR #465.

**Comment by r0ckarong** [comment]
@cwickert You told me it should be broken over into a new line. Do we agree on how this should work?

**Comment by fsundermeyer** [comment]
Fixed with fff76bc
